compose/reference/up.md

```diff
@@ -44,8 +45,8 @@ Builds, (re)creates, starts, and attaches to containers for a service.
 
 Unless they are already running, this command also starts any linked services.
 
-The `docker-compose up` command aggregates the output of each container (essentially running `docker-compose logs -f`). When
-the command exits, all containers are stopped. Running `docker-compose up -d`
+The `docker-compose up` command aggregates the output of each container (essentially running `docker-compose logs --follow`). When
```

Comment:
Do we know if these changes were also made upstream (or are these parts not in the cli help output of compose?)

Reply:
No, there was only a PR to change it here. The usage of docker-compose shows both the short and long options.